### PR TITLE
Update CLI tutorial to use ISS position use case

### PR DESCRIPTION
**Pull request by autophagy**

### CODE REVIEW

File: docs/generate-time-series/cli.rst [hunk 1]
```diff
@@ -32,96 +32,81 @@ Crash is available as `Pip`_ package. `Install`_ it like this:
...
+Using ``curl`` to get the current position of the ISS
+=====================================================
 
-.. code-block:: console
-
-    $ uptime
-    16:58  up 3 days,  5:45, 4 users, load averages: 3.15 4.51 5.06
+You can get telemetry data from `Open Notify`_, a third-party service that
+provides a simple API to consume data from NASA (specifically, the current
+location of the International Space Station). The endpoint for this data is
```
Comment (matthijskrul) [inline docs/generate-time-series/cli.rst:49]: Imho "these data", data is plural. Or "this set of data", etc.

Comment (nomicode) [reply docs/generate-time-series/cli.rst:49]: disagree. data is a singular mass noun! :)

edit: wikipedia seems to back me up re common use, cf. https://en.wikipedia.org/wiki/Data_(word)

**Review by nomicode — APPROVED** [review]
added an edit commit (please rebase, etc, etc!)

changes:

- line wrapping
- switch to using "you" instead of "we"

thank you!! <3

**Comment by autophagy** [comment]
:+1:

**Comment by nomicode** [comment]
@autophagy wanna merge?

**Pull request merged**